packages/shared/src/components/cores/CoreOptionList.tsx

```diff
@@ -28,10 +23,10 @@ export const CoreOptionList = (): ReactElement => {
         prices?.map((price) => {
           return (
             <CoreOptionButton
-              key={price.value}
-              id={price.value}
-              label={price.label}
-              cores={price.coresValue}
+              key={price.priceId}
+              id={price.priceId}
+              label={price.metadata.title}
+              cores={price.metadata.coresValue}
```

Comment:
Are these filled in the db or still from paddle custom data?

Reply:
This should now be maintained in our DB. So that we won't have to update it from Paddle/iOS/ or whichever new store we will work with.

Reply:
yeah perfect